_includes/footer.html

```diff
@@ -1,3 +1,3 @@
 <footer>
-    &copy; {{ site.time | date: '%Y' }} Welcoming Falls Church. All rights reserved. Designed by <a href="https://erikboesen.com">Erik Boesen</a>.
+    &copy; {{ site.time | date: '%Y' }} Welcoming Falls Church. All rights reserved. Designed by <a href="https://erikboesen.com">Erik Boesen</a>. Webmaster Andrew Lester.
```

Comment:
Personally I'd go for "Designed by Erik Boesen, maintained by Andrew Lester" or even just "Erik Boesen & Andrew Lester"

Reply:
If I'm ever to create a website for myself I'll probably change it. I've got to do that soon...